README.md

```diff
@@ -1,3 +1,75 @@
-# Entry point for AGIMUS project demos
+# Agimus Demos
 
-Follow [these steps](doc/docker.md) to set up the docker for the project.
+Agimus Demos is a set of ROS 2 packages used to launch different demos developed for [Agimus Project](https://www.agimus-project.eu/).
+
+Repository is organized as set of demos with increasing complexity. Each folder contains a ROS 2 package launching the demo. README.md of each demo contains description with expected results and instruction how to launch the demo.
+
+All of the demos are launched in a similar manner. For Gazebo simulation use
+```bash
+ros2 launch agimus_demo_<demo-name> bringup.launch.py use_gazebo:=true use_rviz:=true
+```
+For the real robot use
+```bash
+ros2 launch agimus_demo_<demo-name> bringup.launch.py robot_ip:=<robot-ip> use_rviz:=true
+```
+
+## Running demos with Docker
+
+> [!NOTE]
+> This is a recommended installation for this package. You can always install it from source (see below), but to avoid issues with building from source we advise you to use prebuilt docker images, that contains all the dependencies.
+
+> [!CAUTION]
+> Current demos only make use of `agimus_dev_container:humble-devel-control`. There is no need to pull Docker images with vision dependencies.
+
+Docker image is provided in a form of Development Container and can be found at [agimus-project/agimus_dev_container](https://gitlab.laas.fr/agimus-project/agimus_dev_container). Use branch `humble-devel` and refer to the README.md file for more information on how to use it.
+
+This docker has all the dependencies preinstalled and is split into three images: control, vision and vision-cuda. Control image contains dependencies found in [franka.repos](franka.repos) and [control.repos](control.repos). Vision image is not yet supported by this repository.
+
```

Comment:
Steps to install:
- `git clone -b humble-devel https://gitlab.laas.fr/agimus-project/agimus_dev_container.git`
- reopen in container
- `./setup.sh` (???)